quint/src/cliCommands.ts

```diff
@@ -285,157 +285,145 @@ export async function runTests(prev: TypecheckedStage): Promise<CLIProcedure<Tes
   const testing = { ...prev, stage: 'testing' as stage }
   const mainArg = prev.args.main
   const mainName = mainArg ? mainArg : basename(prev.args.input, '.qnt')
-  const main = prev.modules.find(m => m.name === mainName)
 
   const outputTemplate = testing.args.output
 
-  if (!main) {
-    return left({
-      msg: `Module ${mainName} not found`,
-      stage: { ...prev, errors: [] },
-    })
-  } else {
-    const rngOrError = mkRng(prev.args.seed)
-    if (rngOrError.isLeft()) {
-      return cliErr(rngOrError.value, { ...testing, errors: [] })
-    }
-    const rng = rngOrError.unwrap()
+  const rngOrError = mkRng(prev.args.seed)
```

Comment:
The diff in this file is a mess. What I did was simply remove the `main` check, which was a top-level `if`, and then indentation changed. `compileAndTest` now receives `mainName` instead of `main`, so the check is now done inside `compileAndTest`.